medusa/post_processor.py

```diff
@@ -165,101 +146,102 @@ def _compare_file_size(self, existing_file):
             ))
             return self.EXISTS_LARGER if new_size < old_size else self.EXISTS_SMALLER
 
-    def list_associated_files(self, filepath, base_name_only=False, subtitles_only=False, subfolders=False):
+    def list_associated_files(self, filepath, subfolders=None, subtitles_only=None, refine=None):
         """
-        For a given file path search for files in the same directory and return their absolute paths.
+        For a given file path search for associated files and return their absolute paths.
 
-        :param filepath: The file to check for associated files
-        :param base_name_only: list only files with the same basename
-        :param subtitles_only: list only subtitles
-        :param subfolders: check subfolders while listing files
+        :param filepath: path of the file to check for associated files
+        :param subfolders: also check subfolders while searching files
+        :param subtitles_only: list only associated subtitles
+        :param refine: refine the associated files with additional options
         :return: A list containing all files which are associated to the given file
         """
-        files = self._search_files(filepath, subfolders=subfolders, base_name_only=base_name_only)
+        files = self._search_files(filepath, subfolders=subfolders, basename_only=True)
```

Comment:
@medariox this breaks when a .nfo is associated to the .rar base name

As you are only searching files that are equal .mkv base name, it won't find the .nfo as a file